src/modules/pass.js

```diff
@@ -419,6 +429,7 @@ PassFF.Pass = (function () {
                                            || isUrlField(item.key)
                                            || isOtpauthField(item.key));
             item.hasFields = item.children.some(c => allItems[c].isField);
+            item.isHidden = isHidden(item.fullKey);
```

Comment:
While the method is nice for separating concerns, for performance reasons, you could avoid recreating the regex for every single item by creating it once right before this block starting at `allItems`.

Something to the effect of:

```js
var isHiddenRegex = new RegExp(PassFF.Preferences.filterFieldRegex, 'i');
allItems.slice().reverse().forEach(item => {
  // ...
  item.isHidden = isHiddenRegex.test(item.fullKey);
  // ...
});

Reply:
Smart. Something I should have noticed myself. Cheers.